content/docs/concepts/security/index.md

```diff
@@ -135,10 +135,10 @@ Currently we use different certificate key provisioning mechanisms for each scen
 
 1. The above CSR process repeats periodically for certificate and key rotation.
 
-### Node Agent in Kubernetes (in development)
+### Node Agent in Kubernetes
 
-In the near future, Istio will use node agent in Kubernetes for certificate and key provision, as shown in the figure below.
-Note that the identity provision flow for on-premises machines is the same so we only describe the Kubernetes scenario.
+Istio provides the option of using node agent in Kubernetes for certificate and key provisioning, as shown in the figure below.
+Note that the identity provision flow for on-premises machines will be similar in near future, we only describe the Kubernetes scenario here.
```

Comment:
provision -> provisioning
in near -> in the near

Reply:
updated.